Bio/SeqIO/AceIO.py

```diff
@@ -64,23 +63,16 @@ def AceIterator(source):
         consensus_seq_str = ace_contig.sequence
         # Assume its DNA unless there is a U in it,
         molecule_type = "DNA"
-        if "U" in consensus_seq_str:
-            if "T" in consensus_seq_str:
-                # Very odd! Error?
-                alpha = generic_nucleotide
-            else:
-                alpha = generic_rna
-                molecule_type = "RNA"
-        else:
-            alpha = generic_dna
+        if "U" in consensus_seq_str and "T" not in consensus_seq_str:
+            molecule_type = "RNA"
```

Comment:
If it's not written in the file, don't set the `molecule_type`.

Reply:
Conceded on this case, *In the face of ambiguity, refuse the temptation to guess.*
https://www.python.org/dev/peps/pep-0020/